style.css

```diff
@@ -1,464 +1,1083 @@
-  line-height: 24px;
-  list-style: none;
-  padding-bottom: 10px;
-}
-
-.abt-social li {
-  display: inline-block;
-  margin-right: 2%;
-}
-
-.abt-social li img {
-  font-style: normal;
-  font-weight: 500;
-  font-size: 16px;
-  line-height: 24px;
-  color: #7f8cff;
-  text-decoration: none;
-  margin-left: 285%;
-}
-
-.skills {
-  display: flex;
-  flex-direction: column;
-  align-items: start;
-}
-
-.arrow {
-  display: flex;
-  width: 90%;
-  justify-content: space-between;
-  align-items: center;
-  padding: 1rem 0;
-}
-
-.tech {
-  font-style: normal;
-  font-weight: 500;
-  font-size: 15px;
-  line-height: 20px;
-  margin: 15px;
-  color: #253858;
-}
-
-.languages li {
-  background: #f7f7f9;
-}
-
-.contact-container {
-  margin: 0 auto;
-  position: relative;
-  max-height: 800px;
-  margin-bottom: 50px;
-  background-color: white;
-}
-
-.form-container {
-  position: relative;
-  background-color: #6070ff;
-  margin-top: 177px;
-  border-top-left-radius: 80px;
-  display: flex;
-  flex-direction: column;
-}
-
-.back-spot {
-  position: absolute;
-  z-index: 1;
-  top: 21px;
-  right: 0;
-}
-
-.form-bottom {
-  z-index: 99;
-}
-
-.form-bottom h2 {
-  font-style: normal;
-  font-weight: 700;
-  font-size: 40px;
-  line-height: 52px;
-  text-align: center;
-  margin-top: 103px;
-  color: white;
-}
-
-.form-bottom p {
-  font-style: normal;
-  font-weight: 500;
-  font-size: 20px;
-  line-height: 28px;
-  margin-top: 12px;
-  margin-left: 24px;
-  margin-right: 24px;
-  color: white;
-  text-align: center;
-}
-
-.first {
-  margin: 52px 24px 0 24px;
-  background: #fff;
-  border: 1px solid #cfd8dc;
-  border-radius: 8px;
-  width: 327px;
-  padding: 10px;
-}
-
-.first::placeholder {
-  font-style: normal;
-  font-weight: 400;
-  font-size: 17px;
-  line-height: 20px;
-  display: flex;
-  align-items: center;
-  color: #172b4d;
-}
-
-.second {
-  margin: 20px 24px 0 24px;
-  background: #fff;
-  border: 1px solid #cfd8dc;
-  border-radius: 8px;
-  width: 327px;
-  padding: 10px;
-}
-
-.second::placeholder {
-  font-style: normal;
-  font-weight: 400;
-  font-size: 17px;
-  line-height: 20px;
-  display: flex;
-  align-items: center;
-  color: #172b4d;
-}
-
-.message {
-  background: #fff;
-  margin-top: 18px;
-  margin-left: 24px;
-  width: 327px;
-  padding: 15px;
-  border: 1px solid #cfd8dc;
-  border-radius: 8px;
-}
-
-.btn {
-  margin-top: 22px;
-  margin-left: 24px;
-  padding: 12px 16px;
-  background: #fff;
-  border-radius: 8px;
-  border: none;
-  font-style: normal;
-  font-weight: 500;
-  font-size: 17px;
-  line-height: 24px;
-  cursor: pointer;
-  display: flex;
-  align-items: center;
-  text-align: center;
-  letter-spacing: 0.03em;
-  color: #6070ff;
-}
-
-.btn:hover {
-  background-color: #bdbdf1;
-  color: white;
-}
-
-.form-bottom hr {
-  margin: 0 auto;
-  margin-bottom: 8px;
-  width: 125px;
-  height: 5px;
-  background-color: black;
-  margin-top: 36px;
-  border-radius: 10px;
-  border: none;
-}
+* {

```

Comment:
- [ ] As suggested by the previous reviewer you have to make sure the mobile version is working properly as well although it's not specified explicitly in the project requirements, you have to make sure as a developer the new changes you introduce don't break the previous one and as your code reviewer, we are obliged to check that. So please make sure everything looks as it should in the mobile:
     - Here's how it looks now:
        ![firefox_oUd5As6C9m](https://user-images.githubusercontent.com/104968236/230733700-ae92b5a0-e614-4c6b-b2cd-fc14cfb99c80.png)
     - As you can see the project description text is looking very large
        ![firefox_7Wgp7f4WcK](https://user-images.githubusercontent.com/104968236/230733766-e225b08d-7f56-4655-9e10-67f749445902.png)
     - As you can see the project image is misaligned with the description, the image should've been on top.
     ![firefox_KXRS6OXZdM](https://user-images.githubusercontent.com/104968236/230733838-e9a211ca-d021-4d1e-90a4-747fe5b4213b.png)
     - As you can see the "get my resume" button is not looking as it should be
- [ ] So please make sure all these things are working ✔.

Reply:
Solved.